src/adapter/src/session/vars.rs

```diff
@@ -445,6 +445,14 @@ static MOCK_AUDIT_EVENT_TIMESTAMP: ServerVar<Option<mz_repr::Timestamp>> = Serve
     safe: false,
 };
 
+const IS_SUPERUSER: ServerVar<str> = ServerVar {
+    name: UncasedStr::new("is_superuser"),
+    value: "off",
```

Comment:
I'm not a huge fan of using "on" and "off" here, but that's what PostgreSQL does.

Reply:
Is this how all PostgreSQL boolean vars work? Or is this one just super weird?

Reply:
It looks like that's how all boolean vars work. They accepts true/false when setting them, but they print as "on"/"off".
https://www.postgresql.org/docs/current/config-setting.html
> Boolean: Values can be written as on, off, true, false, yes, no, 1, 0 (all case-insensitive) or any unambiguous prefix of one of these.

```
postgres=# SELECT name, setting FROM pg_settings WHERE vartype = 'bool' LIMIT 5;
            name            | setting 
----------------------------+---------
 allow_in_place_tablespaces | off
 allow_system_table_mods    | off
 array_nulls                | on
 autovacuum                 | on
 bonjour                    | off
(5 rows)
```

I'll do some digging and see if there's a setting that Postgres has to change these to true/false.

Reply:
Oh, so then probably this should be a `ServerVar<bool>`, and we should change our output format for bool vars!

Reply:
Good call, looks like that already is how we format bools and I just missed it:

```
materialize=> SHOW sql_safe_updates;
 sql_safe_updates 
------------------
 off
(1 row)
```

Reply:
Updated